spec/user_interface_spec.rb

```diff
@@ -126,6 +129,53 @@
     end
   end
 
+  describe '#add_another_contact?' do
+    it 'prints prompt to user' do
+      input = StringIO.new('y')
+      ui = described_class.new(input, output)
+
+      ui.add_another_contact?
+
+      expect(output.string).to include(described_class::ANOTHER_CONTACT_PROMPT)
+    end
+
+    it 'returns true if user input is y' do
+      input = StringIO.new('y')
+      ui = described_class.new(input, output)
+
+      result = ui.add_another_contact?
+
+      expect(result).to eq(true)
+    end
+
+    it 'returns false if user input is n' do
+      input = StringIO.new('n')
+      ui = described_class.new(input, output)
+
+      result = ui.add_another_contact?
+
+      expect(result).to eq(false)
+    end
+
+    it 'prints error message if incorrect input is given' do
+      input = StringIO.new("e\ny\n")
```

Comment:
`e` is not very descriptive, something like `wrong` or `wrong input` would be more intuitive :+1: